tests/ignite/metrics/test_ssim.py

```diff
@@ -81,7 +81,7 @@ def _test_ssim(y_pred, y, data_range, kernel_size, sigma, gaussian, use_sample_c
         skimg_y,
         win_size=kernel_size,
         sigma=sigma,
-        multichannel=True,
+        channel_axis=-1,
```

Comment:
Regarding to the doc, `channel_axis=-1` is similar to `multichannel=False`.

I don’t understand the reason why you replaced `channel_axis=3`. Could you give some feedback on that please ?

Reply:
https://github.com/scikit-image/scikit-image/issues/4294 
in this discussion they agreed upon,

> multichannel=True becomes channel_axis=-1.
multichannel=False becomes channel_axis=None.

and in the test_ssim.py
```
 y_pred = torch.rand(8, 3, 224, 224, device=device)
 _test_ssim(
        y_pred, y, data_range=1.0, kernel_size=7, sigma=1.5, gaussian=False, use_sample_covariance=True, device=device
    )
```
and inside def _test_ssim():
we are doing, 
`skimg_pred = y_pred.permute(0, 2, 3, 1).cpu().numpy()`

So any ways I do it , whether I do channel_axis = -1 or channel_axis = 3 it would work,
but I think its more professional to write channel_axis = -1 and according to skimage discussion on this topic that I linked, 
multichannel was discontinued for the reason that https://github.com/scikit-image/scikit-image/issues/4294#issue-521910634 
its my first try to contribute, please correct me if I read or interpreted anything wrong,    
I think channel_axis = -1 is what we should go with,
thanks @sdesrozis

Reply:
Ok, according to the discussion, I would prefer having `channel_axis=3` which seems to be a correct usage in our case.

what do you think ?

Reply:
as we used `multichannel = True`  its the hard coded version of `channel_axis = -1`  so that is the reason I changed , anyways its the same , and fixes the warnings and as the members of skiimage treated `multichannel` I think we should treat it the same way so if in future someone refers to it, and does not check `permute(0, 2, 3, 1)` and` (8, 3, 224, 224) `would assume last to represent channel i.e axis = -1

Reply:
IMO the value -1 could be deprecated in the next version since it's like a patch. It's safer to have the correct usage of `channel_axis` especially if you have checked the results are the same.

Note that using correctly this parameter, you would be able to avoid reordering and address the shape (B, C, W, H) with `channel_axis=1`. Could you try this please ? Thanks again for your help !

Reply:
great!  I have done that ... lets deprecate the old permute and use channel_axis.